olive/model/handler/mixin/hf_config.py

```diff
@@ -66,28 +66,31 @@ def _get_hf_model_tokenizer(self, **kwargs):
         # TODO(anyone): only provide relevant kwargs, no use case for now to provide kwargs
         return get_hf_model_tokenizer(self.get_model_path_or_name(), **kwargs)
 
-    def save_metadata_for_token_generation(self, output_dir: str, **kwargs) -> List[str]:
+    def save_metadata_for_token_generation(
```

Comment:
do we need change the signature? I think kwargs should contains these values if user specify them?